Tuesday - Jan 12th, 2016
- I am running python 2.7

- Tried to use pyusb to connect to oscilloscope
	- Hit a wall immediately while trying to list connected devices
	- Apparently I need a "backend"
- I know I used pyvisa before, so now I'm trying that.
	- pyvisa is here - https://pyvisa.readthedocs.org/en/stable/index.html
	- it can also be installed through pip
		- pip install pyvisa
	- Must install NI-VISA
		- Support windows and macOS, but only limited linux distros
		- Slightly worried about incompatibility with ubuntu.
- Searching for solutions to the "cannot find backend" problem in pyusb
	- I think pyusb would be the better option if it worked so that we could use it on multiple os's

- Looking at possibly using usbtmc
	- Works with linux, windows, and macos, but requires pyusb on windows.
	- had to install libusb-win32
		- ran inf_wizard.exe to make a usb device available
	- installed python-usbtmc
	- Trying to run it in ipython
		- Getting a million errors trying to connect to usb device

- pyusb is working now?  I ran the libusb-win32 inf_wizard.exe and now pyusb is working I guess.
- I'll try to see if pyusb will work with the oscilloscope tomorrow or Thursday.


Summary:
python packages installed: pyusb, pyvisa, python-usbtmc
software: ni-visa, libusb-win32